src/Psalm/Internal/Analyzer/IssueData.php

```diff
@@ -154,7 +156,7 @@ public function __construct(
         $this->file_name = $file_name;
         $this->file_path = $file_path;
         $this->snippet = $snippet;
-        $this->selected_text = $selected_text;
+        $this->selected_text = trim($selected_text);
```

Comment:
Trimming here may have far reaching effects - everything that operates on `IssueData` entries is potentially affected, including AfterAnalysis plugins. Could it perhaps be safer to move the trim to where issue is compared to the issue from baseline?

Reply:
Yes, that seems safer - I've moved the `trim`. Tests still pass, including the manual test with applying the patch file. Thanks.